lib/src/main/java/com/diffplug/spotless/generic/IndentStep.java

```diff
@@ -100,6 +100,15 @@ String format(String raw) {
 					++contentStart;
 				}
 
+				// detect multi-line comments
+				boolean multiLineComment = false;
+				if (contentStart < raw.length() && raw.charAt(contentStart) == '*') {
+					multiLineComment = true;
+					if (contentStart == lineStart) {
+						++numSpaces; // multi-line comment without a leading space
+					}
```

Comment:
The step must not insert a space that wasn't already present, it should only preserve existing spaces.

```java
boolean mightBeMultilineComment = contentStart < raw.length() && raw.charAt(contentStart) == '*';
```

Reply:
> it should only preserve existing spaces

Got it - apparently I misunderstood the responsibilities of `IndentStep`.